### ISSUE TITLE
Merge into Ember Data

### ISSUE THREAD

**Issue by sandstrom** [danielspaniel/ember-data-change-tracker#52]
Has there been any discussions with the Ember Data team about moving this functionality into Ember Data? 

Basically, to use this addon as the starting point for an RFC that builds on https://github.com/emberjs/rfcs/pull/21, but probably supersedes it in a few ways.

It would be very useful! 🎈

**Comment by danielspaniel** [comment]
This could happen one day @sandstrom , but I am not really pushing for or against it. Just sitting on the fence waiting for the record data things to happen and waiting for ED to evolve a bit this year. People are aware of the need to dirty track relationships ( for sure ) so if you want to hustle that process along, your more than welcome. But I not sure if it is a big priority for ED team.

**Comment by sandstrom** [comment]
Makes sense, the record data changes seems to be moving along at a good pace (https://github.com/emberjs/data/issues/5618). Better to help bringing that into stable before any work on integrating better dirty tracking.

Awesome addon! 🏅 Perhaps it may grow into ED at some point.

**Comment by danielspaniel** [comment]
thanks for the good vibes @sandstrom .. keeps me => 🕺
